### PR TITLE
Python2 and Python3 compatible

### PR DESCRIPTION
**Pull request by ghost**
I made the script compatible with python2 and python3.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea/
```
Comment (hnsl) [inline .gitignore]: Please remove this file. You can put this in your global .gitignore. See: https://help.github.com/articles/ignoring-files/

File: unox.py [hunk 2]
```diff
@@ -20,14 +20,20 @@
...
 import fsevents
-import urllib
 import traceback
 import signal
 
+# Import depending on python version
+if sys.version_info.major < 3:
+    from urllib import quote, unquote
+else:
+    from urllib.parse import quote, unquote
```
Comment (hnsl) [inline unox.py:31]: Have you tested that the script still works on both python2 and python3 with this change?

**Review by hnsl — COMMENTED** [review]
Thank you for your pull request. Please address comments and I will approve.

File: unox.py (hunk 2 above)
Comment (ghost) [reply unox.py:31]: Jup. I've tested it manually.

I've started the script in a python2 and another python3 virtualenv.

I've also changed the first line to `#!/usr/bin/env python2` and `#!/usr/bin/env python3` and it detects filesystem changes in my docker-sync setup. 

😃

**Comment by ghost** [comment]
This is ready for another review.

**Comment by ghost** [comment]
I'm creating a new pull request with based on the updated master with the watchdog changes.

**Pull request closed without merging**